src/components/registration/steps.jsx

```diff
@@ -226,6 +226,7 @@ module.exports = {
     ChoosePasswordStep: intl.injectIntl(React.createClass({
         getDefaultProps: function () {
             return {
+                studentUsername: null,
```

Comment:
I think it might good to go with `username` for now – I think it'd be helpful to have a bit more context than `notEquals`, and I'm not sure what else, in other contexts (like reset password or something), we would have that value be different from `username`.